maud/src/lib.rs

```diff
@@ -7,13 +7,6 @@
 
 #![doc(html_root_url = "https://docs.rs/maud/0.22.0")]
```

Comment:
this could be `https://docs.rs/maud/0/maud/` so you don't have to update it as often

Reply:
Yeah, I considered that but I'd rather keep it precise – I don't want someone's old build to have dead links just because the API changed.

A few respectable crates (e.g. [`syn`](https://docs.rs/syn/1.0.42/src/syn/lib.rs.html#253)) do the same thing so we're in good company.

Reply:
Ahh interesting. Wonder if upcoming intra-doc links will make this less necessary. Anyway I just hate having to remember this type of thing xD